### ISSUE TITLE
Tabbing without selecting a single-select has TypeError /  null dereference

### ISSUE THREAD

**Issue by timdiggins** [yairEO/tagify#1296]
## Prerequisites

- [x] I am running the latest version
- [x] I checked the documentation and found no answer
- [x] I checked to make sure that this issue has not already been filed

### 💥 Demo Page

https://yaireo.github.io/tagify/#section-mode-select

### Explanation

- **What is the expected behavior?**
I can click (or tab) into a single select field, NOT select anything and (with the drop down still showing) tab to next field without error

- **What is happening instead?**

When I tab, an error is raised (and tab action does not occur)

- **What error message are you getting?**

```js
jQuery.tagify.min.js:27 Uncaught (in promise) TypeError: Cannot read properties of null (reading 'value')
    at _.getMappedValue (jQuery.tagify.min.js:27:12074)
    at jQuery.tagify.min.js:27:7002
getMappedValue @ jQuery.tagify.min.js:27
(anonymous) @ jQuery.tagify.min.js:27
Promise.then (async)
onKeyDown @ jQuery.tagify.min.js:27
```

**Comment by yairEO** [comment]
Thanks for reporting I am on it.

**Comment by yairEO** [comment]
fixed in [`v4.22`](https://github.com/yairEO/tagify/releases/tag/v4.22.0)
